website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md

```diff
@@ -9,70 +9,211 @@ description: >
+  <br>
+  <p>It accepts an `Array` to turn off multiple IDs:</p>
+  <pre>
+  Selenium::WebDriver.logger.ignore(%i[jwp_caps pause pauses])</pre>
+  <br>
+  <p>To turn off all *deprecation* notices:</p>
+  <pre>
+  Selenium::WebDriver.logger.ignore(:deprecations)</pre>
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+**Useful Information**
+
+This is the default level where Selenium logs things that users should be aware of but do not need to take actions on.
+It presents information such as requests and responses between driver and server, payload, etc.
+
+Different languages have different level to log information.
+
+{{< tabpane langEqualsHeader=true text=true >}}
+  {{< tab header="Java" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+  Logs useful information at level: <code>:info</code>
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+  Logs useful information at level: <code>FINER</code>
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+**Debugging Details**
+
+The debug log level is used for information that may be needed for diagnosing issues and troubleshooting problems.
+
+{{< tabpane langEqualsHeader=true text=true >}}
+  {{< tab header="Java" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+  Logs debugging details at level: <code>:info</code> and <code>:debug</code>
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+  Logs debugging details at level: <code>FINER</code> and <code>FINEST</code>
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+### 3. Log Output:
+Logs can be displayed on `stdout` or stored in a file.
+
+{{< tabpane langEqualsHeader=true text=true >}}
+  {{< tab header="Java" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+  <p>By default, logs are sent to the console in `stdout`.</p>
+  <p>To store the logs in a file:</p>
+  <pre>
+  Selenium::WebDriver.logger.output = '/path/to/selenium.log'</pre>
+  <br>
+  <p>Sample Output:</p>
+  <blockquote>
```

Comment:
This formatting is hard to read, I think we need to treat this as a codeblock instead of a blockquote so that it uses a mono format.